### PR TITLE
Lizard nerfs

### LINKED ISSUE #14909: lizards can use abductor weaponry
wew
probably not actually a feature

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
They can no longer use alien guns.

Fixes #14909

### CODE REVIEW

**Comment by hornygranny** [comment]
>this actually got merged in its original state

**Comment by Zilenan91** [comment]
fun removal

File: code/game/gamemodes/miniantags/abduction/abduction_gear.dm [hunk 1]
```diff
@@ -316,7 +316,7 @@
 /obj/item/device/firing_pin/alien/pin_auth(mob/living/user)
 	if(ishuman(user))
 		var/mob/living/carbon/human/H = user
-		if(H.dna.species.id == "human") //stealth lizard buff go
+		if(H.dna.species.id != "abductor")
```
Comment (neersighted) [inline code/game/gamemodes/miniantags/abduction/abduction_gear.dm]: AYY

Comment (PKPenguin321) [inline code/game/gamemodes/miniantags/abduction/abduction_gear.dm]: LMAO

Comment (tedward1337) [inline code/game/gamemodes/miniantags/abduction/abduction_gear.dm]: Fucking KEK

**Pull request merged**